model/resource/k8s.yaml

```
@@ -33,6 +33,11 @@ groups:
     attributes:
       - ref: k8s.pod.uid
       - ref: k8s.pod.name
+      - id: labels
+        type: template[string]
+        brief: >
+          The labels placed on the Pod, the `<key>` being the label name, the value being the label value.
+        examples: ['k8s.pod.labels.app=my-app,k8s.pod.labels.mycompany.io/arch=x64,k8s.pod.labels.data=']
```

```suggestion
        examples: ['k8s.pod.labels.app=my-app','k8s.pod.labels.mycompany.io/arch=x64','k8s.pod.labels.data=']
```
Is the last label in the example intentionally empty?

---

I think it is: https://github.com/open-telemetry/semantic-conventions/pull/494#discussion_r1394760997